recipe/meta.yaml

```
@@ -10,14 +10,14 @@ source:
 
 build:
   number: 0
-  skip: true  # [not linux64]
+  noarch: generic
```

Think we still need a skip here as there are no `linux_ppc64le` builds

```suggestion
  skip: true  # [not ppc64le]
```

---

So don't make it `noarch` at all? https://github.com/conda-forge/nvidia-gds-feedstock/issues/1

---

Yeah unfortunately. This is because we would want to add `__linux` to `requirements/run`, but we can't do that if `linux_ppc64le` doesn't also work

---

Oh right, thanks!